crm_claim_code/models/crm_claim.py

```diff
@@ -20,12 +20,16 @@ class CrmClaim(models.Model):
     @api.model
     def create(self, vals):
         if vals.get('code', '/') == '/':
-            vals['code'] = self.env['ir.sequence'].get('crm.claim')
+            vals['code'] = self.env['ir.sequence'].next_by_code('crm.claim')
         return super(CrmClaim, self).create(vals)
 
-    @api.one
+    @api.multi
     def copy(self, default=None):
         if default is None:
             default = {}
-        default['code'] = self.env['ir.sequence'].get('crm.claim')
-        return super(CrmClaim, self).copy(default)
+        results = self.env['crm.claim']
+        for record in self:
+            default['code'] = self.env['ir.sequence'].next_by_code('crm.claim')
```

Comment:
Better to use:
```python
default.setdefault('code', self.env['ir.sequence'].next_by_code('crm.claim'))
```
So if the caller had explicitly set a default for `code`, it would be kept.